content/en/users/datatransfer/api/_index.md

```diff
@@ -0,0 +1,283 @@
+---
+title: "API"
+linkTitle: "API"
+type: docs
+description: "Documentation for EGI Data Transfer API"
+weight: 30
+---
+
+## Overview 
+
+The EGI Data Transfer service offers API both for Users and Admins, in this section 
+we are focusing on the User API.
+2 APIs are available to users:
```

Comment:
Maybe it's more developers than users?

Reply:
I addd this part on the API as for for DataHub we did the same, we can keep this or remove the API section from both

Reply:
I think it's nice/useful to have access to information about API.
I was more willing to say that it may have been phrased, but users is fine:
```
Two APIs are available to developers:
```